agent/engine/docker_task_engine.go

```diff
@@ -469,6 +594,9 @@ func (engine *DockerTaskEngine) synchronizeState() {
 		engine.saveTaskData(task)
 	}
 
+	// Before starting managedTask goroutines, pre-allocate resources for already running
+	// tasks in host resource manager
```

Comment:
Not exactly "already running", more like tasks that have progressed beyond the resource consumption check

Reply:
Yes, will update these in follow up PR